crates/ironclaw_gateway/static/index.html

```
@@ -426,6 +426,7 @@ <h2 data-i18n="restart.title">Restart IronClaw Instance</h2>
           <button class="settings-subtab" data-settings-subtab="skills" data-i18n="tab.skills">Skills</button>
           <button class="settings-subtab" data-settings-subtab="users" data-i18n="settings.users">Users</button>
           <button class="settings-subtab" data-settings-subtab="tools" data-i18n="settings.tools">Tools</button>
+          <button class="settings-subtab" data-settings-subtab="trace-commons" data-i18n="settings.traceCommons">Trace Commons</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)